src/pages/Scholarships.jsx

```diff
@@ -21,11 +24,23 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function ScholarshipsPage() {
+  const classes = useStyles();
   const [scholarships, setScholarships] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState();
   const [sortField, setSortField] = useState('deadline');
   const [sortDir, setSortDir] = useState('asc');
+  const history = useHistory();
+  const location = useLocation();
+  const setFoo = (val) => {
```

Comment:
You can call this `setSort` and then use this directly in FilterBar.jsx

Reply:
Hm I am not using it directly in the FilterBar.jsx instead in FilterDropdown.jsx, as it was kind of repetitive if you see in the previous commits.